packages/sdk/src/swap/types.ts

```diff
@@ -125,6 +125,22 @@ export type SwapStatusResponse = CommonStatusFields &
         broadcastRequestedAt: number;
         broadcastRequestedBlockIndex: string;
       }
+    | {
+        state: 'BROADCASTED';
+        swapId: string;
+        depositAmount: string;
+        depositReceivedAt: number;
+        depositReceivedBlockIndex: string;
+        intermediateAmount: string | undefined;
+        swapExecutedAt: number;
+        swapExecutedBlockIndex: string;
+        egressAmount: string;
+        egressScheduledAt: number;
+        egressScheduledBlockIndex: string;
+        feesPaid: SwapFee[];
+        broadcastRequestedAt: number;
+        broadcastRequestedBlockIndex: string;
+      }
```

Comment:
I think this can just be folded into the `BROADCAST_REQUESTED`, i.e.: `state: 'BROADCAST_REQUESTED' | 'BROADCASTED'`